vector-im/element-x-android
# [Create room] Select members before creating a room (UI for selection)

## Type of change


## Content

Adding a new `selectusers` module which contains api to use when searching users (either a single user or multiple users) is needed.
Adding a new screen to allow selection of multiple users before creating a new room.

## Motivation and context

Closes #108 

## Screenshots / GIFs



<img src="https://user-images.githubusercontent.com/46314705/225660382-90201e7c-bf18-476c-bfc3-1db67953cb43.gif" width=300 />

## Tests



- On the room list screen, press the start new message button
- Press the "New room" button
- Press the search bar
- Enter a text matching a matrix id pattern
- Select the result item
- Check the user is added into the list selection
- Add another user
- Check the new user is added at the start of the list selection

## Tested devices

- OS version(s): Android 11

## Checklist



- [x] Pull request includes a new file under ./changelog.d.
